### PR TITLE
Lesson3 Homework

### PR DESCRIPTION
**Pull request by Lilit-Lewonyan**

### CODE REVIEW

File: src/Lesson3.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        int[] arrtask2 = new int[1000];
+        int j = 0;
+        System.out.println("Output of Task 2: ");
+        while (j < 1000) {
+            arrtask2[j] = j + 1;
+            if ((1000 - arrtask2[j]) % 100 == 0 && arrtask2[j]!=1)
+                System.out.println(" " + arrtask2[j] + " ");
+            else
+                System.out.print(" " + arrtask2[j] + " ");
+            j++;
```
Comment (Karengrigoryan060) [inline src/Lesson3.java:21]: why we need this?

File: src/Lesson3.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        for (int i = 0; i < 2; i++) {
+            System.out.println("element " + i + "= " + arrtask1[i]);
+        }
+
+        //Task 2: Create an array and fill it with numbers from 1:1000.
+        int[] arrtask2 = new int[1000];
+        int j = 0;
+        System.out.println("Output of Task 2: ");
+        while (j < 1000) {
+            arrtask2[j] = j + 1;
```
Comment (Karengrigoryan060) [inline src/Lesson3.java:16]: wouldn't be better to use for instead of while?

File: src/Lesson3.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            arrtask3[e] =k;
+            if ((40 - e) % 10 == 0 && e!= 0)
+                System.out.println(" " + arrtask3[e] + " ");
+            else
+                System.out.print(" " + arrtask3[e] + " ");
+            k++;
+        }
+
+
+        }
```
Comment (Karengrigoryan060) [inline src/Lesson3.java:39]: didn't get

File: src/Lesson3Homework2.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        //Task 6:Create an array and fill it. Print count of elements which can be divided by 2.
+        int[] arrtask6 = {66,55,728,448,1599,11,12,19,2756,22,30,38,471,45,798,44,159,118,123,18};
+        int k=0;
+        int count =0;
+        while (k<20){
+            if(arrtask6[k]%2==0)
+                count++;
+            k++;
+        }
+        System.out.println("count of elements which can be divided by 2 = "+ count );
```
Comment (Karengrigoryan060) [inline src/Lesson3Homework2.java:37]: why so hard?

File: src/Lesson3.java (hunk 1 above)
Comment (Lilit-Lewonyan) [reply src/Lesson3.java:21]: I  print  the result and did  that in each line print less then 100 elements that it would be easy to read.

**Pull request merged**